3d_segmentation/unetr_segmentation_3d.ipynb

```diff
@@ -0,0 +1,4317 @@
+{
```

Comment:
<p>Could you please help review the output of this cell? It's some info of your own hardware.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Fixed.